sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_BuildCustomModel.md

```diff
@@ -22,6 +22,11 @@ var client = new DocumentModelAdministrationClient(new Uri(endpoint), credential
 ## Build a custom model
 
 Build custom models to extract text, field values, selection marks, and table data from documents. Custom models are trained with your own data, so they're tailored to your documents.
+
+When building a custom model, you'll need to choose a build mode that's appropriate to the type of documents you have:
+- Template build mode: recommended when the custom documents all have the same layout. Fields are expected to be in the same place across documents. Build time tends to be considerably shorter than the neural build mode.
+- Neural build mode: recommended when custom documents have different layouts. Fields are expected to be the same but they can be placed in different positions across documents.
+
```

Comment:
I prefer to have a link here, but that's what we have for now. This is my understanding based on our conversations with the service team, so I used this text pretty much everywhere. Feel free to let me know if there's any fix/rewording you think it's valid here.

Reply:
I looked back at chats and some more info was [shared](https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1643335864812?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1643313562835&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1643335864812) but it was prefaced with "FYI only" so not sure we want to add anything from there. I agree that keeping it brief and linking to an aka.ms will work for now